src/components/Navbar.tsx

```diff
@@ -1,24 +1,22 @@
-export const Navbar = () => {
-  return (
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">Home</a>
+export const Navbar = () => (
+  <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <a className="navbar-item" href="#/">Home</a>
 
-          <a
-            aria-current="page"
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
+        <a
+          aria-current="page"
+          className="navbar-item has-background-grey-lighter"
+          href="#/people"
```

Comment:
Use React Router's Link component instead of an anchor tag for internal navigation.

Reply:
Thanks, there was a bigger problem here. Navbar was not used. 😅